pom.xml

```diff
@@ -894,6 +894,45 @@ under the License.
                 </plugins>
             </build>
         </profile>
+        <profile>
```

Comment:
this is clever! let's add a note / readme to the pubsub example that specifies that it needs to be built with this new profile

Reply:
I added en example on ReadMe for Pubsub